libraries/angularjs/src/components.js

```diff
@@ -81,6 +81,34 @@ const ComponentWithProps = {
   }
 };
 
+const ComponentWithInheritance = {
```

Comment:
I don't know much about angularjs but looking at what's already here, i think the component needs to be added here:
https://github.com/webcomponents/custom-elements-everywhere/blob/c5c71ed0e069c645c8db359a70c2f05aa88ea78e/libraries/angularjs/src/app.module.js#L1-L21

needs to import `ComponentWithInheritance` declared here, and needs
```js
.component('compWithInheritance', ComponentWithInheritance)
```

and the top of this file also has import lines, which probably needs this added
```js
import 'ce-with-inheritance';
```

Reply:
Hmm.... it's working w/o that, but I'm not sure what that means.